src/client/pages/ThemaPagina/ThemaPagina.tsx

```diff
@@ -76,6 +80,20 @@ export default function ThemaPagina({
             </Grid.Cell>
           )}
 
+          {!!buttonLinkItems?.length && (
+            <Grid.Cell span="all">
+              {buttonLinkItems.map(({ to, title }) => (
+                <Button
+                  variant="secondary"
+                  key={to}
+                  onClick={() => history.push(to)}
+                >
+                  {title}
+                </Button>
+              ))}
+            </Grid.Cell>
+          )}
```

Comment:
Wat is dit?

Reply:
vervangen, misschien niet zon goede toevoeging voor alleen dit geval